HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -173,6 +173,11 @@ def self.apply_header(hpxml_header, epw_file)
       hpxml_header.temperature_capacitance_multiplier_isdefaulted = true
     end
 
+    if hpxml_header.defrost_model_type.nil?
+      hpxml_header.defrost_model_type = HPXML::AdvancedResearchDefrostModelTypeStandard
+      hpxml_header.defrost_model_type_isdefaulted = true
+    end
```

Comment:
Only do this if there's a relevant heat pump in HPXML. No point to have the defrost type added to `in.xml` if it's not relevant.